references/plugins/references.go

```diff
@@ -689,8 +707,8 @@ type HELMSchema struct {
 	Schemas *openapi3.Schema
 }
 
-// GenerateHELMProperties get all properties of a HelmCategory type capability
-func (ref *ParseReference) GenerateHELMProperties(ctx context.Context, capability *types.Capability) ([]HELMReference, []ConsoleReference, error) {
+// GenerateHelmAndKubeProperties get all properties of a Helm/Kube Category type capability
+func (ref *ParseReference) GenerateHelmAndKubeProperties(ctx context.Context, capability *types.Capability) ([]HELMReference, []ConsoleReference, error) {
 	cmName := fmt.Sprintf("%s%s", types.CapabilityConfigMapNamePrefix, capability.Name)
 	var cm v1.ConfigMap
 	helmRefs = make([]HELMReference, 0)
```

Comment:
```suggestion
	helmRefs = make([]JSONReference, 0)
```
please change to a more general name